cli/bin/setup

```diff
@@ -7,15 +7,4 @@ if [ $? -gt 0 ]; then
   exit 1
 fi
 yarn install
-(cd ../exo-add && bin/setup && echo 'exo-add √')
-(cd ../exo-clone && bin/setup && echo 'exo-clone √')
-(cd ../exo-create && bin/setup && echo 'exo-create √')
-(cd ../exo-deploy && bin/setup && echo 'exo-deploy √')
-(cd ../exo-lint && bin/setup && echo 'exo-lint √')
-(cd ../exo-run && bin/setup && echo 'exo-run √')
-(cd ../exo-setup && bin/setup && echo 'exo-setup √')
-(cd ../exo-sync && bin/setup && echo 'exo-sync √')
-(cd ../exo-test && bin/setup && echo 'exo-test √')
-(cd ../exosphere-shared && bin/setup && echo 'exosphere-shared √')
-(node_modules/.bin/build)
-(./bin/build)
+node_modules/.bin/build
```

Comment:
I think the issue before was that the cli repo uses the rest of the `exo` commands. So when a PR is made with only changes to cli, Morula will only run tests (and `setup`) for cli. That means `setup` will not be run for the rest of the `exo` commands and their `dist` folders will not be available to the cli feature tests.